apm-server/templates/ingress.yaml

```diff
@@ -12,23 +13,25 @@ metadata:
     {{- range $key, $value := .Values.labels }}
     {{ $key }}: {{ $value | quote }}
     {{- end }}
```

Comment:
ingress annotation can be used for other things than `className`, it shouldn't be removed.